Formula/auditbeat.rb

```diff
@@ -49,6 +49,8 @@ def install
         --path.logs #{var}/log/auditbeat \
         "$@"
     EOS
+
+    generate_completions_from_executable(bin/"auditbeat", "completion", shells: [:bash, :zsh])
```

Comment:
Looks like the file isn't executable yet (typically we can rely on brew handling this for us after building):
```suggestion
    chmod 0555, bin/"auditbeat"
    generate_completions_from_executable(bin/"auditbeat", "completion", shells: [:bash, :zsh])
```